package.json

```diff
@@ -194,27 +196,38 @@
     "precommit": "yarn run test && lint-staged",
     "lint-js": "eslint --ignore-path .gitignore --ignore-pattern \"!**/.*\" .",
     "lint-css": "stylelint \"src/**/*.{css,less,styl,scss,sass,sss}\"",
-    "lint": "yarn run lint-js && yarn run lint-css",
+    "lint": "yarn run lint-js && npm run lint-css",
```

Comment:
Any reason you need `npm` for this versus `yarn`? I'd lean toward changing all commands to `npm run` if one of them doesn't work properly with `yarn`.

Reply:
> Yeah, in my initial fiddling around to get CircleCI to work, I concluded
that the sane thing was to keep everything `npm` except for `yarn install`

@svoynow Yes but you only changed half of `lint` and `fix`, and no other commands.